static/app/utils/customMeasurements/customMeasurementsProvider.tsx

```diff
@@ -6,20 +6,24 @@ import {Client} from 'sentry/api';
 import {normalizeDateTimeParams} from 'sentry/components/organizations/pageFilters/parse';
 import {t} from 'sentry/locale';
 import {Organization, PageFilters} from 'sentry/types';
+import {CustomMeasurementCollection} from 'sentry/utils/customMeasurements/customMeasurements';
 import handleXhrErrorResponse from 'sentry/utils/handleXhrErrorResponse';
-import {MeasurementCollection} from 'sentry/utils/measurements/measurements';
 import useApi from 'sentry/utils/useApi';
 
 import {
   CustomMeasurementsContext,
   CustomMeasurementsContextValue,
 } from './customMeasurementsContext';
 
+type measurementsMetaResponse = {
```

Comment:
```suggestion
type MeasurementsMetaResponse = {
```

I think if we define this as a type it should be capitalized